### ISSUE TITLE
Cannot use 'in' operator to search for '<name of view>' in undefined

### ISSUE THREAD

**Issue by danielhq** [Dogstudio/highway#29]
Hello

I'm trying to use this plugin but after I import the script (using both npm and the minified version) 

```javascript
highway.js:424 Uncaught TypeError: Cannot use 'in' operator to search for 'home' in undefined
```
in my main js file i have:

```javascript
       import Highway from '@dogstudio/highway';
 	const H = new Highway.Core();
```
I followed the example and copied and pasted the HTML from the documentation.

What am i doing wrong?

**Comment by ThaoD5** [comment]
Hello @danielhq !

The problem is probably coming from the fact that you're not having any renderer ( which is supposed to be fine ), can you, just to be sure, maybe create a renderer for `home` like in the example just to make sure this is the correct issue here ? 

If it is, we're going to have a look at this as soon as possible to fix this problem because your code is supposed to work like that :-) 

Thanks for your help and bringing this issue to us

Thao

**Comment by Anthodpnt** [comment]
@danielhq This is now fixed in the v2.0.7, please update and tell us if it's now OK for you as well.
